src/views/FirstAssessmentPage.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="container">
```

Comment:
Classes are CapitalCasing. Can you change the classes in this file to CapitalCasing?